Source/WebGPU/WebGPU/RenderBundleEncoder.mm

```diff
@@ -291,11 +291,14 @@ - (instancetype)initWithICB:(id<MTLIndirectCommandBuffer>)icb containerBuffer:(i
     return addResource(resources, mtlResource, [[ResourceUsageAndRenderStage alloc] initWithUsage:MTLResourceUsageRead renderStages:stage entryUsage:BindGroupEntryUsage::Input binding:BindGroupEntryUsageData::invalidBindingIndex resource:resource]);
 }
 
-bool RenderBundleEncoder::executePreDrawCommands(bool passWasSplit)
+bool RenderBundleEncoder::executePreDrawCommands(bool passWasSplit, uint32_t firstInstance, uint32_t instanceCount)
 {
     if (!isValid())
         return false;
 
+    if (checkedSum<uint64_t>(instanceCount, firstInstance) > std::numeric_limits<uint32_t>::max())
```

Comment:
what is the result of checkedSum when it overflows?

Reply:
It can't overflow currently because they are both `uint32_t` being passed to a `uint64_t` called to `checkedSum`:
   `uint32_t firstInstance, uint32_t instanceCount`